safe/gui/tools/help/peta_bencana_help.py

```diff
@@ -81,9 +82,9 @@ def content():
         'This tool requires a working internet connection and fetching '
         'data will consume your bandwidth.'))
     tips.add(m.Link(
-        'https://petajakarta.org/banjir/en/data/',
+        production_api['help_url'],
         text=tr(
-            'Downloaded data is copyright the PetaJakarta contributors'
+            'Downloaded data is copyright the PetaBencana contributors'
```

Comment:
Oops, I'm commenting right after you comment, so I didn't see your comment there. Either is fine, if we can be consistent, we can easily change it later :D.